Dockerfile

```diff
@@ -24,25 +26,22 @@ RUN buildDeps='libmagic-dev' && \
     apt-get purge -y --auto-remove $buildDeps && \
     rm -Rf /var/lib/apt/lists/* /var/cache/apt/* ~/.gemrc ~/.bundle
 
-# use the default config (or else the app won't start)
-# most non-system-related settings can be changed in the app anyway
-RUN mv config/app_config.yml.SAMPLE config/app_config.yml
-
-# compile assets with temporary mysql server
+# compile assets with temporary mysql server and generate crontab
 RUN export DATABASE_URL=mysql2://localhost/temp && \
     export SECRET_KEY_BASE=thisisnotimportantnow && \
     export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
     apt-get install -y mysql-server && \
     /etc/init.d/mysql start && \
+    cp config/app_config.yml.SAMPLE config/app_config.yml && \
     bundle exec rake db:setup assets:precompile && \
-    rm -Rf tmp/* && \
+    rm -Rf config/app_config.yml tmp/* && \
     /etc/init.d/mysql stop && \
-    rm -Rf /run/mysqld /tmp/* /var/lib/mysql /var/log/mysql* && \
+    rm -Rf /run/mysqld /tmp/* /var/tmp/* /var/lib/mysql /var/log/mysql* && \
     apt-get purge -y --auto-remove mysql-server && \
-    rm -Rf /var/lib/apt/lists/* /var/cache/apt/*
-
-RUN bundle exec whenever >crontab
+    rm -Rf /var/lib/apt/lists/* /var/cache/apt/* && \
+    \
+    bundle exec whenever >crontab
```

Comment:
i'd move the crontab to the previous `RUN`, so only the temoprary mysql stuff is in its own `RUN`